src/app/pages/interest-form/interest-form.hbs

```diff
@@ -0,0 +1,201 @@
+<h1>OpenStax College Interest Form</h1>
+<p>
+    Thank you for your interest in OpenStax College textbooks! Please complete the form below so that we may provide
+    you with the information you need to start saving your students thousands of dollars each semester.
+</p>
+<form accept-charset="UTF-8" action="/adoption_interests" class="new_adoption_interest" method="post">
+    <div>
+        <input name="utf8" type="hidden" value="✓">
+        <input name="authenticity_token" type="hidden" value="Xi14qRzCbSy2zkso1VEjKU4bEUfFWOrhWbQnh7tlO3c=">
```

Comment:
This appears to be a CSRF token generated by rails, probably on the form you were duplicating.  It wouldn't have any meaning here, and would need to be randomly generated (and verified) by the server.

I'm all for using a CSRF token, but we need to have server support to do so.  So I think we either need to coordinate with @rich-hart and @mwharrison or just remove this line for the time being.

It looks like this also [exists on the adoption form page](https://github.com/openstax/os-webview/pull/6/files#diff-985278373ff40a89fe057052763be39fR18), although I missed it before it got merged.